### PR TITLE
refactor: truly migrate to grafana 9

### PR DESCRIPTION
**Pull request by Z10N0110**
1. Add new guest/guest_nice mode CPU panel for VMs
2. Real CPU modes utilization query in CPU panels
2. Switch to `irate` fuction for range-vector queries, see more @ https://www.robustperception.io/irate-graphs-are-better-graphs/

### CODE REVIEW

**Comment by rfmoz** [comment]
Please, could you pass every change splitted in small commits? It's hard to follow the 1k line changes. Thanks,

**Comment by rfmoz** [comment]
Finally I managed to check the diff and it looks great. Thanks!

**Comment by AustinLeath** [comment]
thanks!
